### PR TITLE
15 make icons + text on contact page hyperlinked

### PR DESCRIPTION
**Pull request by NeerajMehta04**
Contacts_icon+text link

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -31,22 +31,43 @@ <h1>Get in touch with us</h1>
                     
                 </div>
                 <div class="getintouch-icons">
+
                     <div class="getintouch-icon-ig icon-row">
+                        <a href="https://www.instagram.com/restore.trdxals/?hl=en" >                        
                         <img src="Images/Contact/Social-media-icons/logo-instagram-2.png" alt="">
+                        </a>
+                        <a href="https://www.instagram.com/restore.trdxals/?hl=en" >
```
Comment (quarc0o) [inline contact.html:39]: add: target=”_blank” to the anchor tags so that the link opens in a new window

**Review by quarc0o — CHANGES_REQUESTED** [review]
I made some comments 😁

**Comment by quarc0o** [comment]
Wrong branch 😅

**Comment by quarc0o** [comment]
Wrong branch 😅

**Review by quarc0o — DISMISSED** [review]
Wrong branch 😅

**Review by ReStore-TRD — APPROVED** [review]

**Pull request merged**